types/V1.mli

```diff
@@ -457,13 +457,49 @@ module type IP = sig
 
 end
 
+module type ARP = sig
+  include DEVICE
+
+  type ipaddr
+  type buffer
+
+  (** Type of the result of an ARP query.  One of `Ok Macaddr.t (for successful
+      queries) or `Timeout (for attempted queries that received no response). *)
+  type result = [ `Ok of Macaddr.t | `Timeout ]
+
+  (** Prettyprint cache contents *)
+  val prettyprint : t -> string
```

Comment:
indeed, it would be better to have `val pp: Format.formatter -> t -> unit` here. I'll slowly deprecate all the other `t -> string` signatures.